exporter/azuremonitorexporter/traceexporter.go

```diff
@@ -713,9 +713,7 @@ func newTraceExporter(config *Config, transportChannel transportChannel, logger
 
 	exp, err := exporterhelper.NewTraceExporter(
 		config,
-		exporter.pushTraceData,
-		exporterhelper.WithTracing(true),
-		exporterhelper.WithMetrics(true))
```

Comment:
Q: we don't need these anymore? Are they automatically enabled by the new observability package?

Reply:
Correct: there were a few misusages of them disabling metrics or traces when we didn't want them.